README.md

```diff
@@ -7,367 +7,506 @@
 
+
 ### Implementation decisions
-Given the aforementioned requirements, there are some aspects of the implementation that were chosen, and are
-particularly under consideration:
-* Aside from its configuration, the IoT Agent Lib is considered to be stateless. To be precise, the library
-mantains a state (the list of entities/devices whose information the agent can provide) but that state is considered
-to be transient. It's up to the particular implementation of the agent to consider whether it should have a persistent
-storage to hold the device information (so the internal list of devices is read from a DB) or to register the devices
-each time a device sends a measure. To this extent, two flavours of the Device Registry has been provided: a transient
-one (In-memory Registry) and a persistent one (based in MongoDB).
-* The IoT Agent does not care about the origin of the data, its type or structure. The mapping from raw data to the
-entity model, if there is any, is a responsability of the particular IoT Agent implementation, or of another third
-party library.
 
-##  Build and Install
+Given the aforementioned requirements, there are some aspects of the
+implementation that were chosen, and are particularly under consideration:
+
+-   Aside from its configuration, the IoT Agent Lib is considered to be
+    stateless. To be precise, the library mantains a state (the list of
+    entities/devices whose information the agent can provide) but that state is
+    considered to be transient. It's up to the particular implementation of the
+    agent to consider whether it should have a persistent storage to hold the
+    device information (so the internal list of devices is read from a DB) or to
+    register the devices each time a device sends a measure. To this extent, two
+    flavours of the Device Registry has been provided: a transient one
+    (In-memory Registry) and a persistent one (based in MongoDB).
+-   The IoT Agent does not care about the origin of the data, its type or
+    structure. The mapping from raw data to the entity model, if there is any,
+    is a responsability of the particular IoT Agent implementation, or of
+    another third party library.
+
+## Install
+
+Information about how to configure the Library can be found at the corresponding
+section of the
+[Installation & Administration Guide](https://iotagent-node-lib.readthedocs.io/en/latest/installationguide).
 
-Information about how to configure the Library can be found at the corresponding section of the [Installation & Administration Guide](doc/installationguide.md).
+## Usage
 
-This library has no packaging or build processes. Usage of the library is explained in the [User & Programmers Manual](doc/usermanual.md).
+This library has no packaging or build processes. Usage of the library is
+explained in the
+[User & Programmers Manual](https://iotagent-node-lib.readthedocs.io/en/latest/usermanual).
```

Comment:
Same comment to https://github.com/telefonicaid/iotagent-node-lib/pull/690/files#r234587237 here.

Reply:
Fixed -  e12947e